### PR TITLE
docs: add section about getting logs not available on stdout

### PR DESCRIPTION
**Pull request by kkujawa-sumo**
###### Description

Fill in your description here.

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/docs/Best_Practices.md [hunk 1]
```diff
@@ -549,3 +550,30 @@ kube-prometheus-stack:
 
 **NOTE** We observed that changing this value increases metrics loss during prometheus resharding,
 but the traffic is much better balanced between Fluentds and Prometheus is more stable in terms of memory.
+
+## Get logs not available on stdout
+
+When logs from Pod are not available on stdout [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
+helps with collecting them using standard logging pipeline.
```
Comment (pmalek-sumo) [inline deploy/docs/Best_Practices.md:557]:
```suggestion
When logs from a Pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
can help with collecting them using standard logging pipeline.
```

File: deploy/docs/Best_Practices.md [hunk 2]
```diff
@@ -549,3 +550,30 @@ kube-prometheus-stack:
...
+To tail logs using Tailing Sidecar Operator the file with logs needs to be accessible through volume
+because the volume provides the access to the log file and this volume is mounted to sidecar container.
```
Comment (pmalek-sumo) [inline deploy/docs/Best_Practices.md:559]:
```suggestion
To tail logs using Tailing Sidecar Operator the file with those logs needs to be accessible through a volume
mounted to sidecar container.
```

File: deploy/docs/Best_Practices.md (hunk 1 above)
Comment (kkujawa-sumo) [reply deploy/docs/Best_Practices.md]: Corrected

File: deploy/docs/Best_Practices.md (hunk 2 above)
Comment (kkujawa-sumo) [reply deploy/docs/Best_Practices.md]: Corrected

**Review by pmalek-sumo — APPROVED** [review]

**Review by astencel-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]

**Comment by perk-sumo** [comment]
Can we also reference it [here](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/release-v2.1/deploy/docs/Best_Practices.md#add-a-local-file-to-fluent-bit-configuration) so that people know that they can use the `tailing sidecar` instead of changing the `fluent bit` config?

**Comment by sumo-drosiek** [comment]
@perk-sumo We should mention that, but IMHO this is different usecase (maybe some common single cases): file from node vs file from container

File: deploy/docs/Best_Practices.md [hunk 3]
```diff
@@ -580,3 +584,31 @@ kube-prometheus-stack:  # For values.yaml
   prometheus:
     prometheusSpec:
       scrapeInterval: '1m'
+```
+
+## Get logs not available on stdout
+
+When logs from a Pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
```
Comment (sumo-drosiek) [inline deploy/docs/Best_Practices.md:591]:
```suggestion
When logs from a pod are not available on stdout, [Tailing Sidecar Operator](https://github.com/SumoLogic/tailing-sidecar)
```
?

**Review by pmalek-sumo — APPROVED** [review]

File: deploy/docs/Best_Practices.md (hunk 3 above)
Comment (kkujawa-sumo) [reply deploy/docs/Best_Practices.md:591]: I don't know how it should be written, In k8s docs Pod starts with capital letter, in other places it is written either as "Pod" or "pod"

Comment (sumo-drosiek) [reply deploy/docs/Best_Practices.md:591]:
```
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' pod ' **/**.md | wc -l
55
➜  sumologic-kubernetes-collection git:(drosiek-otc) ✗ grep ' Pod ' **/**.md | wc -l
7
```

I have no idea neither, by statistically we want to use `pod` :sweat_smile:

**Review by perk-sumo — APPROVED** [review]

**Pull request merged**